Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstoryStreamingAPI/GoogleChronicleBackstoryStreamingAPI.py

```diff
@@ -906,15 +1040,16 @@ def stream_detection_alerts_in_retry_loop(client: Client, initial_continuation_t
 def main():
     """PARSE AND VALIDATE INTEGRATION PARAMS."""
     # initialize configuration parameter
-    proxy = demisto.params().get('proxy')
-    disable_ssl = demisto.params().get('insecure', False)
+    params = remove_space_from_args(demisto.params())
```

Comment:
Was there an issue which causes this to be needed?

Reply:
Hi @amshamah419, thanks for reviewing the PR. We have added it to follow the best practices.